### PR TITLE
Spanish translation update for 1.4

### PR DESCRIPTION
**Pull request by TomMalbran**
The long awaited spanish translation update is here… And this one was a hard one…

### CODE REVIEW

**Comment by abose** [comment]
As release,1.4 is targeted for today or tomorrow, merging this today. 
@jbalsas Could you review this.

**Comment by TomMalbran** [comment]
Thanks. @busykai can also review this :)

File: src/nls/es/strings.js [hunk 1]
```diff
@@ -192,12 +193,13 @@ define({
     "FIND_IN_FILES_MORE_THAN"           : "Más de ",
     "FIND_IN_FILES_PAGING"              : "{0}&mdash;{1}",
     "FIND_IN_FILES_FILE_PATH"           : "<span class='dialog-filename'>{0}</span> {2} <span class='dialog-path'>{1}</span>",
+    "FIND_IN_FILES_INDEXING"            : "Indexando archivos para la Búsqueda instantánea...",
```
Comment (busykai) [inline src/nls/es/strings.js]: Probably better use `\u2026` for the ellipsis.

File: src/nls/es/strings.js [hunk 2]
```diff
@@ -317,6 +319,7 @@ define({
     "CMD_FILE_SAVE_ALL"                   : "Guardar todo",
     "CMD_FILE_SAVE_AS"                    : "Guardar como\u2026",
     "CMD_LIVE_FILE_PREVIEW"               : "Vista previa dinámica",
+    "CMD_TOGGLE_LIVE_PREVIEW_MB_MODE"     : "Habilitar la ista previa dinámica experimental",
```
Comment (busykai) [inline src/nls/es/strings.js]: ista -> Vista.

File: src/nls/es/strings.js [hunk 3]
```diff
@@ -582,7 +593,16 @@ define({
...
+    "ERROR_CLTOOLS_RMFAILED"                    : "No se pudó eliminar el enlace simbólico existente <code>/usr/local/bin/brackets</code>.",
+    "ERROR_CLTOOLS_MKDIRFAILED"                 : "No se pudó crear el directorio <code>/usr/local/bin</code>.",
+    "ERROR_CLTOOLS_LNFAILED"                    : "No se pudó crear elenlace simbólico <code>/usr/local/bin/brackets</code>.",
+    "ERROR_CLTOOLS_SERVFAILED"                  : "Error interno.",
+    "ERROR_CLTOOLS_NOTSUPPORTED"                : "El acceso directo por línea de comandos no es compatible con este sistema operativo.",
+    "LAUNCH_SCRIPT_CREATE_SUCCESS"              : "¡Éxito! Ahora puedes abrir fácilmente {APP_NAME} desde la línea de comandos: <code>brackets myFile.txt</code> para abrir un archivo o <code>brackets myFolder</code> para cambiar de proyecto. <br/><br/><a href='https://github.com/adobe/brackets/wiki/Command-Line-Arguments'>Aprende más</a> acerca de como usar {APP_NAME} desde la línea de comandos.",
```
Comment (busykai) [inline src/nls/es/strings.js]: extra space before `<br />`

File: src/nls/es/strings.js [hunk 4]
```diff
@@ -590,6 +610,14 @@ define({
...
+    "HEALTH_DATA_PREVIEW"                       : "Reporte de estadísticas de {APP_NAME}",
+    "HEALTH_DATA_PREVIEW_INTRO"                 : "<pPara poder mejorar {APP_NAME},  enviamos periódicamente limitadas estadísticas <strong>anónimas</strong> a Adobe acerca de cómo se utiliza {APP_NAME}. Esta información ayuda a priorizar características a desarrollar, encontrar errores y problemas de usabilidad. <a href='https://github.com/adobe/brackets/wiki/Health-Data'>Aprende más sobre el Reporte de estadísticas de {APP_NAME}</a> y como beneficia a la comunidad de {APP_NAME} mientras protege tu privacidad.</p><p>A continuación hay una vista previa de la información que será enviada en el siguiente Reporte de estadísticas <em>si</em> está habilitado.</p>",
```
Comment (busykai) [inline src/nls/es/strings.js]: `<p` -> `<p>` at the very beginning.

File: src/nls/es/strings.js [hunk 5]
```diff
@@ -611,7 +639,7 @@ define({
...
+    "DETECTED_EXCLUSION_INFO"                   : "{APP_NAME} se encontró con problemas procesando: <span class='dialog-filename'>{0}</span>.<br><br>Este archivo no volverá a ser procesado para las sugerencias de código, saltar a la definición o para la edición rápida. Para reactivar este archivo, abra el archivo <code>.brackets.json</code> en su proyecto y edite <code>jscodehints.detectedExclusions</code><br><br>Esto es probablemente un error en {APP_NAME}. Si puede proporcionar una copia de este archivo, por favor <a href='https://github.com/adobe/brackets/wiki/How-to-Report-an-Issue'>envíe un informe</a> con un vínculo a dicho archivo.",
```
Comment (busykai) [inline src/nls/es/strings.js]: `<br>` -> `<br />` in all occurrences. (even though it comes from before.)

File: src/nls/es/strings.js [hunk 6]
```diff
@@ -623,7 +651,117 @@ define({
...
+    "DESCRIPTION_DETECTED_EXCLUSIONS"                : "Una lista de los archivos que se ha detectado que causan que Tern no funcione correctamente",
+    "DESCRIPTION_INFERENCE_TIMEOUT"                  : "La cantidad de veces después de la cual Tern dejara de intentar entender los archivos",
+    "DESCRIPTION_SHOW_ERRORS_IN_STATUS_BAR"          : "true para mostrar errores en la barra de estado",
+    "DESCRIPTION_QUICK_VIEW_ENABLED"                 : "true para habilitar la Vista rápida",
+    "DESCRIPTION_EXTENSION_LESS_IMAGE_PREVIEW"       : "true para mostrar vistas previas de imágenes para URLs sin extensión",
+    "DESCRIPTION_THEME"                              : "Seleccionar un tema para {APP_NAME}",
+    "DESCRIPTION_USE_THEME_SCROLLBARS"               : "true para habilitar la barra de desplazamiento customizada",
+    "DESCRIPTION_LINTING_COLLAPSED"                  : "true para colapsar el panel de inspección de código",
+    "DESCRIPTION_FONT_FAMILY"                        : "Cambiar la fuente",
+    "DESCRIPTION_FONT_SIZE"                          : "Cambiar el tamaño de la fuente; e.j, 13px",
```
Comment (busykai) [inline src/nls/es/strings.js]: `e.j` -> `ej.`

**Comment by busykai** [comment]
@TomMalbran, there's some stuff worth correcting. If you want me to create a separate PR, let me know.

**Comment by TomMalbran** [comment]
@busykai Thanks for the review. I will try to make a PR soon. But I am doing other stuff. If you want, you could make it, and I can merge it. If not I'll try to make it later today

File: src/nls/es/strings.js (hunk 5 above)
Comment (busykai) [inline src/nls/es/strings.js]: This one is withdrawn. We should really use `<br>` since it's not XHTML and it's not served and parsed as such.

**Pull request merged**